staging/src/k8s.io/apiserver/pkg/endpoints/filters/authentication.go

```diff
@@ -99,22 +47,15 @@ func WithAuthentication(handler http.Handler, auth authenticator.Request, failed
 			req = req.WithContext(authenticator.WithAudiences(req.Context(), apiAuds))
 		}
 		resp, ok, err := auth.AuthenticateRequest(req)
+		defer recordAuthMetrics(resp, ok, err, apiAuds, authenticationStart)
 		if err != nil || !ok {
-			if err != nil {
-				klog.Errorf("Unable to authenticate the request due to an error: %v", err)
```

Comment:
we lost this log line, right? is that intentional?

Reply:
It was not intentional.
Restored the log statement.